src/utils/sort.js

```diff
@@ -0,0 +1,33 @@
+export default (type, direction) => {
+  let dirFactor = direction === 'ascending' ? 1 : -1
+  switch(type){
+  case 'string':
+    return (a,b) => {
+      let aName = a.toLowerCase()
+      let bName = b.toLowerCase()
+      if (aName < bName) {
+        return dirFactor
+      }
+      if (aName > bName) {
+        return -1*dirFactor
+      }
+      return 0
+    }
+  case 'date':
+    return (a,b) => {
+      let aDate = new Date(a)
```

Comment:
`const` here would be fine as we're not reassigning these variables within the scope of this function